packages/main/src/themes/TableRow.css

```diff
@@ -35,3 +36,48 @@
 	font-family: "72override", var(--sapFontFamily);
 	font-size: var(--sapFontSize);
 }
+
+.ui5-table-multi-select-cell {
```

Comment:
Somehow the currently implemented pressed, hover states differ from the openui5 sample:
https://openui5nightly.hana.ondemand.com/entity/sap.m.Table/sample/sap.m.sample.Table

In the openui5, 
the Table rows in MultiSelect Mode don't have hover and active states at all
the Table rows in SingleSelectMaster Mode don't have the active state (although it seems odd not to have it)

Could you check the VD and if you have to fix something?

Reply:
SingleSelectMaster: https://openui5nightly.hana.ondemand.com/entity/sap.m.Table/sample/sap.m.sample.TableDnD
MultiSelect:
https://openui5nightly.hana.ondemand.com/entity/sap.m.Table/sample/sap.m.sample.TableVerticalAlignment

I think that they are similar or I miss something?

Reply:
The rows in https://openui5nightly.hana.ondemand.com/entity/sap.m.Table/sample/sap.m.sample.TableVerticalAlignment example has type="Navigation" and I was testing with a Table with rows with type="Inactive" which is the default type. We don't have a a type here for the TableRow as of now. So, we have to decide which one to implement, as far I can see you implemented the "Active/Navigation" type. The "Inactive" one is the default, but the "Active" looks better.

In the example below if you press down the row and hold, you don't get the dark blue active state,
because the type is Inactive
https://openui5nightly.hana.ondemand.com/entity/sap.m.Table/sample/sap.m.sample.TableDnD


Leave this one for now, I will discuss this detail with the steholders.